pkg/osutil/osutil.go

```diff
@@ -115,6 +123,14 @@ func CopyDir(
 	return nil
 }
 
+func validateAllVariablesSubstituted(s string) error {
```

Comment:
💡 Suggestion: what does `s` mean here?, if we are using some kind of meaningful input string lets logical Renae that variable for better readability.

Reply:
agreed i've updated to fileContent